test/Conversion/TorchToLinalg/view.mlir

```diff
@@ -0,0 +1,83 @@
+// RUN: torch-mlir-opt <%s -convert-torch-to-linalg -split-input-file -verify-diagnostics | FileCheck %s
+
+// -----
+
+// CHECK-LABEL:   func.func @torch.aten.view$twotothree(
+// CHECK-SAME:  %[[arg0:.*]]: !torch.vtensor<[3,2],f32>) -> !torch.vtensor<[2,3],f32> {
+// CHECK:    %[[ZERO:.*]] = torch_c.to_builtin_tensor %[[arg0]] : !torch.vtensor<[3,2],f32> -> tensor<3x2xf32>
+// CHECK:    %[[int3:.*]] = torch.constant.int 3
+// CHECK:    %[[int2:.*]] = torch.constant.int 2
+// CHECK:    %[[ONE:.*]] = torch.prim.ListConstruct %[[int2]], %[[int3]] : (!torch.int, !torch.int) -> !torch.list<int>
+// CHECK:    %[[TWO:.*]] = torch_c.to_i64 %[[int2]]
```

Comment:
This is a really non-standard way of naming FileCheck variables. Can you look in the surrounding files and match that style?